_topic_maps/_topic_map.yml

```
@@ -1261,6 +1261,12 @@ Topics:
   - Name: Disaster recovery considerations
     File: nbde-disaster-recovery-considerations
     Distros: openshift-enterprise,openshift-origin
+- Name: zero trust workload identity manager for Red Hat OpenShift
+  Dir: zero_trust_workload_identity_manager
+  Distros: openshift-enterprise
+  Topics:
+  - Name: zero-trust workload identity manager for Red Hat OpenShift overview
```

Name of the topic is "zero trust" (with no hyphen), but name of the first topic is "zero-trust" (with hyphen). Based on what's already in the docs with "Zero trust networking" I would assume it should be without, but I am not 100% sure:

```suggestion
  - Name: Zero trust workload identity manager for Red Hat OpenShift overview
```